bin/precice-config-visualizer

```diff
@@ -24,7 +24,7 @@ def parseXML(content):
 
 
 def parseXMLFile(file):
-    return parseXML(open("precice.xml", "rb").read())
+    return parseXML(open("precice-config.xml", "rb").read())
```

Comment:
This was not really an intended change, we need to parse here the input argument so that the config file name can be specified.

Reply:
I would postpone the general handling to a later PR. If this change was not intended, do you think it should be reverted? I am a bit confused why a specific name is here anyway.

Reply:
This was previously dead code. Now it has been revived. In my opinion, we should not postpone it because people cannot visualize other configs any more. The solution is probably also very simple and maybe @fsimonis knows it straight away.

Reply:
This was just a testing function and should have been removed.